meetings/2022-06-30.md

```diff
@@ -39,38 +42,99 @@
+* CB: I'd like add re implementation, I have the demo implementation for Web Push. This works in a demo-way but there is still the issue how we model the subscription resource (Subscription resource as a container #[36](https://github.com/solid/notifications/issues/36) ). Resolving this issue will help to adapt my implementation.
+* SC: You should insist on feedback. If I miss in the agenda, propose as topic or use Gitter chat.
+* RG: will review after meeting.
 
 
 ### Discovery of Notification Subscription Metadata
 URL: https://github.com/solid/notifications/issues/58
 
 * SC: There is now `solid:storageDescription` in Solid Protocol ED ( https://github.com/solid/specification/pull/416 , https://solidproject.org/ED/protocol#server-storage-description )
-
-
+* SC: This adresses some of the considerations wrt #58, how much we rely on the Solid Protocol and how much the Notifications Protocol covers.
+* SC: There might be related comments in #76.
+* SC: Discovering the features that are available gives a good idea of what is missing. It doesn't matter where that place for discovery is, but now we have a reliable place where this information lives with the `solid:storageDescription`.
+* SC: If the Notifications Protocol relies on that feature, then it can say what the channel/subscription information should be available there.
+* CB: Maybe a naive question: Why do we not rely on the .meta resource? 
+* SC: I think the URL for that is implementation specific - was it NSS? So we need a FYN type of approach, typically with a link relation to find that.
+* CB: Isn't that also in the Link-Header? 
+* SC: Oh, I think you are describing the `describedby` relation which some implementations use `.meta` as the resource. That's related. That relation can be used on any resource about itself / quote, unquote metadata, but the storageDescription is only about the storage description in which the storage that the resource is in.
+* CB: I see, I thought that in such a meta resource, one could announce that for this resource such notifications would be possible, but if notifications are tied storage, then this makes more sense.
+* SC: Yea, that's part of the considerations in #58 e.g., endpoint per storage for all resources or unique endpoint per resource. Different ways to go at it.. we need to understand/weigh it out.
+* SC: I don't think it makes a whole lot difference what the design is for clients. But matters for servers.
+* CB: right.
+* RG: Has there been any discussion on which way you want to go?
+* SC: I think we somehow defaulted to storage because that seemed "good enough" coverage. It would make the wiring of that information per resource more complicated, whether that's done by out of band by the server/owner or some specific mechanism that allows clients ot do it. It gets complicated for sure ot assign a different channel description info per resource.
```

Comment:
```suggestion
* SC: I think we somehow defaulted to storage because that seemed "good enough" coverage. It would make the wiring of that information more complicated per resource, whether that's done by out of band by the server/owner or some specific mechanism that allows clients to do it. It gets complicated for sure to assign a different channel of description info per resource.
```